modules/ROOT/pages/bolt/message.adoc

```diff
@@ -844,21 +862,44 @@ If a `TELEMETRY` message contains a value that is not a valid `api` value or is
 [source, bolt]
 ----
 C: TELEMETRY 2
+S: FAILURE {"neo4j_code":"Neo.ClientError.Request.Invalid","message":"Message of type TelemetryMessage cannot be handled by a session in the NEGOTIATION state.","gql_status":"50N42","description":"error: general processing exception - unexpected error. Unexpected error has occurred. See debug log for details."}
+----
+
+NOTE: This error still as unexpected (`50N42`) since it didn't get migrate yet to GQL standard.
```

Comment:
This requires a whole lot of background knowledge in how we handle the GQL error migration. I'm in favor of not going into this here. Either make up a status code or don't comment on it, I'd say. It's just an example and no guarantee that the server does actually send these exact values.